modules/acquia_cms_common/src/Commands/AcmsCommands.php

```diff
@@ -286,6 +286,7 @@ function ($key, $cellData, FormatterOptions $options, $rowData) {
       return $result;
     }
     $this->output()->writeln('Starter-kit is not installed yet.');
+    return FALSE;
```

Comment:
Revert this change. Not required

Reply:
It is required as this method returns data, other it will throw "return statement is missing" error in drupal-check.

Reply:
Already pushed into develop branch if you revert this change and check then it won't throw any error, also remove BOOL from return type of starterkit method

Reply:
removed.